pkg/bundle/client.go

```diff
@@ -155,6 +160,41 @@ func (bc *bundleClient) CreateClusterNamespace(ctx context.Context, clusterName
 	return nil
 }
 
+func (bc *bundleClient) CreateClusterConfigMap(ctx context.Context, clusterName string) error {
+	name := auth.ConfigMapName
+	namespace := api.PackageNamespace + "-" + clusterName
+	key := types.NamespacedName{
+		Name:      name,
+		Namespace: namespace,
+	}
+
+	cm := &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+	}
+	err := bc.Get(ctx, key, cm)
+	if err == nil {
+		return nil
```

Comment:
This portion is similar to the function CreateClusterNamespace already in the file. So whats going on here is we are checking if the Namespace/Config Map already exists. If it does we dont need to do anything when reconciling here. Thus a success and no error. So going into the next line of the code we check if the error is type IsNotFound. If it is thats fine and we can create. Otherwise its an actual error and we need to exit the function with error.

Reply:
Can you add that as a comment in the code?

Reply:
Added